lang/csharp/src/apache/test/AvroGen/AvroGenSchemaTests.cs

```diff
@@ -619,7 +619,10 @@ public void NotSupportedSchema(string schema, Type expectedException)
                 string schemaFileName = Path.Combine(outputDir, $"{uniqueId}.avsc");
                 System.IO.File.WriteAllText(schemaFileName, schema);
 
-                Assert.That(AvroGenTool.GenSchema(schemaFileName, outputDir, new Dictionary<string, string>(), false), Is.EqualTo(1));
+                // We now support unknown logical types
+                //Assert.That(AvroGenTool.GenSchema(schemaFileName, outputDir, new Dictionary<string, string>(), false), Is.EqualTo(1));
```

Comment:
The commented out lines could be removed

Reply:
removed